### ISSUE TITLE
[3.3.5][6.x] Nagrand Arena Quest

### ISSUE THREAD

**Issue by DearAdmin** [TrinityCore/TrinityCore#15479]
Hello staff,

Nagrand arena quest is bugged?

so when we are in party if one player accept arena quest, other group members can't get quest in npc, quest go hiding.

Regards,

**Comment by TrinityCoreBot** [comment]
This issue is not considered valid because of the following reasons:
- No proper commit hash (revision)

Please read http://bit.ly/tc-issuetracker-and-you . Thanks.

**Comment by tkrokli** [comment]
1. You have not told us your core revision hash.
2. You have not described or linked the event or quest ID.
3. Your post does not contain enough description to find out when or where the problem manifests.

**Comment by DearAdmin** [comment]
TrinityCore rev. f50df7282018 2015-08-24 14:52:18 +02000 ( 3.3.5 branch) (Win64, Release)

1) http://wotlk.openwow.com/quest=9962 = only one player can get it in group  so other players cant get it because quest are removing from npc  , so can't go other quests

2) http://wotlk.openwow.com/quest=9967

3) http://wotlk.openwow.com/quest=9970

4) http://wotlk.openwow.com/quest=9972

5) http://wotlk.openwow.com/quest=9973

6) http://wotlk.openwow.com/quest=9977

7) http://wotlk.openwow.com/quest=9962

**Comment by tkrokli** [comment]
OK, that's better. That is what your original first post should look like. Please remember it to future issues.

**Comment by Killyana** [comment]
And how it works in retail, any player can start any event?

**Comment by tkrokli** [comment]
From reading the comment pages on wowhead, I see that it is suggested that the tank picks up the quest first to get initial aggro, so I presume that 

I don't remember the exact details, but from what I remember when doing those quests in the TBC expansion, there is a very short time frame for accepting the quest and also completing the quest after each fight. The most common scenario used to be that a lot of matches had to be redone for most, if not all players in the group. The most common cause would be that the players were not on the same quest at the same time. Maybe I was victim to a bug at that time, but I did not pick up that quest often.

Looking at http://www.wowhead.com/quest=9962/the-ring-of-blood-brokentoe#comments:id=12506 I see the comment `Note: Make sure all quest members have turned in previous quest before starting.`
I would recommend reading the wowhead pages for this questline, especially the highest rated posts. Maybe it could be an issue of the players not having done the required preliminary quest?

**Comment by dr-j** [comment]
Well what should happen is first player in party takes quest and then when that player takes quest other players in party should be asked if they also want to start te quest provided they are elegible (ie at same point in quest chain (ie convoy quest) this did work idk if still does.

**Comment by Saben65** [comment]
As I recall, you have from the time you kill the creature, till the time the next person started the next quest. Once a party member starts the quest, everyone who is able to accept the quest should receive a UI prompt asking if they want to start the quest as well.

**Comment by Killyana** [comment]
With the actual scripting engine the only change we can do is to add conditions to avoid running on quest accept events if there's one of the summoned npcs around, and remove the change of the npc flag.

**Comment by DearAdmin** [comment]
only problem is, when you accept other players are cant see quest in npc,

btw its same issue wih zuldrak arena quest, only 1 player can get, If you complete quest.. then other player can accept,, weird..

**Comment by iarkoff** [comment]
https://github.com/TrinityCore/TrinityCore/issues/15479#issuecomment-139078714

the popup asking if the players want to accept the quest appers, but clicking "Accept" doesn't put the quest in your questlog, even if you are elegible.

it's the same thing for the nagrand and zul'drak quests.

it forces the party to kill the same npc multiple times (one for each party member) when only one should count for everyone.

i don't know if it's a general problem with "chain-quests" or just these two

**Comment by DearAdmin** [comment]
so

**Comment by dr-j** [comment]
I think this was broken by some commit quiet a while ago to attempt to fix quest sharing which it may have fixed issue there but broke this type of quest share, cant remember exactly which commit but ye it used to work believe has been broken for several months and its not just these quests its all convoy quests.

**Comment by DearAdmin** [comment]
no more answer ? :o:

**Comment by Rushor** [comment]
fixed with https://github.com/TrinityCore/TrinityCore/commit/3c3cde028cbcab64d463707e192fc333d8cadda0 ?

**Comment by Aokromes** [comment]
No answer after more than 4 months, if this bug is still valid open a new ticket.
